### ISSUE TITLE
Time to move on... migrate to Wordpress??

### ISSUE THREAD

**Issue by therourke** [MitchellMcKenna/LifePress#61]
I have been using Sweetcron/Lifepress now for [8 years or so](http://machinemachine.net/stream/). But it's time to move on. I have completely changed the front end of my wordpress website, and the sweetcron backend is starting to get far too unwieldy.

I don't want to give up entirely on the concept of Lifepress. I used it as an extended archive that linked in to the tags of my wordpress blog.

I was wondering if anyone would be interested in helping me write some scripts to migrate the Lifepress mysql over to wordpress? I don't have the skills to do this, but I am pretty sure it wouldn't be too difficult. 

There's only a small amount of information necessary to move over: Lifepress > Wordpress equivalent

- Title > title
- tags > tags
- content > content
- linked URL > using a Wordpress plugin like ['Page Links To'](https://wordpress.org/plugins/page-links-to/) it would be easy to replicate external URLs
- post image > featured image

RSS feed cron jobs can be dealt with by plugins like [WP RSS Aggregator](https://en-gb.wordpress.org/plugins/wp-rss-aggregator/)

Wordpress Categories could take the place of site IDs, and then it wouldn't be too difficult to create Wordpress themes that altered the output depending on post category. 

[My new wordpress front page](http://machinemachine.net/) at the moment looks *something* like I imagine a Lifepress port to look. Each front page Featured Image / post holder post would link externally, and there would be Categories for things like twitter, youtube videos etc.

Basically, I am just throwing the idea out there, since the Lifepress development hasn't moved very far in 5 years or so, it seems like a good idea to create tools to port it over. I seem to remember someone many years ago in the Sweetcron community talking about doing this.

Any thoughts?
